cluster/local/local.sh

```diff
@@ -23,6 +23,10 @@ function copy_image_to_cluster() {
 
 case "${1:-}" in
   up)
+    # We'll use locally cached image instead of having it downloaded by kind
+    # cluster.
+    docker pull "gcr.io/kubernetes-helm/tiller:${HELM_VERSION}"
+    copy_image_to_cluster "gcr.io/kubernetes-helm/tiller:${HELM_VERSION}" "gcr.io/kubernetes-helm/tiller:${HELM_VERSION}"
```

Comment:
I think this is a no-op in the local case, it is just tagging the image with its own name